cli/deno_error.rs

```diff
@@ -185,6 +186,19 @@ impl GetErrorKind for hyper::Error {
   }
 }
 
+impl GetErrorKind for ReadlineError {
+  fn kind(&self) -> ErrorKind {
+    use ReadlineError::*;
+    match self {
+      Io(_err) => unimplemented!(),
+      Eof => ErrorKind::UnexpectedEof,
+      Interrupted => unimplemented!(),
+      Utf8Error => unimplemented!(),
+      Errno(_err) => unimplemented!(),
```

Comment:
Not sure about how to handle these other errors. I would just wait until we hit them..

Reply:
This variant exists only on unix (type is nix::Error), so this will probably cause a build error on windows.
We do implement GetErrorKind for nix::Error too, so on unix `Errno(err) => err.kind()` should work.

Reply:
ok, i made it `#[cfg(unix)]`